src/app/ApplicationTemplate.Shared.Views/App.xaml.cs

```diff
@@ -33,23 +32,11 @@ public App()
 
 	public Window CurrentWindow { get; private set; }
 
-	protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
-	{
-		InitializeAndStart();
-	}
-
-	/*
-	protected override void OnActivated(IActivatedEventArgs args)
-	{
-		// This is where your app launches if you use custom schemes, Universal Links, or Android App Links.
-		InitializeAndStart(args);
-	}
-	*/
-
-	private void InitializeAndStart(IActivatedEventArgs args = null)
+	protected override void OnLaunched(LaunchActivatedEventArgs args)
 	{
 #if WINDOWS
```

Comment:
missed that on other place, but I think we need to wrap those if with `//+:cnd:noEmit`

Reply:
@jeanplevesque @MatFillion Am I right? I never really understood this part. I guess it's part of the `dotnet new template`

Reply:
I added it for now.

Reply:
Yes, you need to wrap any pre-compiler instructions like `#if` around `//-:cnd:noEmit` `//+:cnd:noEmit` in order to keep it when materializing the dotnet template.
If you don't, the dotnet templating engine will drop that code.

Reply:
Can we had a space between `//` and the text? Would remove VS warning and propositions.

Reply:
not sure but you can test by running the dotnet new and looking at the resulting code.

Reply:
Or by looking at the generated code of the GeneratedApp from the pipelines. All its code should be in the build artifacts.

Reply:
Will try in a separate Pull Request.